### PR TITLE
fix log stdlibadapter when prefixed

### PR DESCRIPTION
**Pull request by soven**

### CODE REVIEW

File: log/stdlib.go [hunk 1]
```diff
@@ -49,6 +52,11 @@ func MessageKey(key string) StdlibAdapterOption {
 	return func(a *StdlibAdapter) { a.messageKey = key }
 }
 
+// Prefix sets value of the prefix for the actual log message. By default, there's no prefix.
```
Comment (peterbourgon) [inline log/stdlib.go:55]: What does setting the prefix do?

Comment (soven) [reply log/stdlib.go:55]: It allows client code to use the adapter for stdlib logger which is set by the prefix. There is an example in [the issue](https://github.com/go-kit/kit/issues/1035).

Comment (peterbourgon) [reply log/stdlib.go:55]: Can you please add some more detail to the doc comment?

File: log/stdlib.go [hunk 2]
```diff
@@ -26,9 +27,11 @@ func (w StdlibWriter) Write(p []byte) (int, error) {
 // messages, and place them under relevant keys.
 type StdlibAdapter struct {
 	Logger
-	timestampKey string
-	fileKey      string
-	messageKey   string
```
Comment (whiteboy19787) [inline log/stdlib.go:31]: I'm not sure what to do this is new to me

File: log/stdlib_test.go [hunk 3]
```diff
@@ -72,6 +72,56 @@ func TestStdLibAdapterExtraction(t *testing.T) {
 	}
```
Comment (whiteboy19787) [inline log/stdlib_test.go:72]:
```suggestion
	}grab bin
```

File: log/stdlib.go (hunk 1 above)
Comment (soven) [reply log/stdlib.go]: sure. done

**Comment by soven** [comment]
looks like something wrong with the CI

File: log/stdlib.go [hunk 4]
```diff
@@ -49,6 +52,21 @@ func MessageKey(key string) StdlibAdapterOption {
...
+// there's no prefix.
+//
+// When you create a stdlib Logger, you can define a log prefix like this:
+//
+//		logger := log.New(w, "some prefix", log.LstdFlags)
+//
+// To correctly extract stdlib logger fields, you should set the same prefix
+// by Prefix stdlib adapter option. Also you can set joinPrefixToMsg if you need
+// to join the prefix to extracted message fields. For example, you can need it
+// when using log.Lmsgprefix flag.
```
Comment (peterbourgon) [inline log/stdlib.go:65]:
```suggestion
// Prefix configures the adapter to parse a prefix from stdlib log events. If
// you provide a non-empty prefix to the stdlib logger, then your should provide
// that same prefix to the adapter via this option.
//
// By default, the prefix isn't included in the msg key. Set joinPrefixToMsg to
// true if you want to include the parsed prefix in the msg.
```

**Comment by peterbourgon** [comment]
Thank you!

**Pull request merged**